k8s/configuration/secret.yaml

```diff
@@ -0,0 +1,92 @@
+# All secret data should be base64 encoded before embedding them here by
+# using `echo "secret string" | base64 -w 0 > secret.string.b64` and then
+# copy the resulting value here.
+# Ref: https://kubernetes.io/docs/concepts/configuration/secret/
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-agent-api-key
+  namespace: default
+type: Opaque
+data:
+  # This is the API Key obtained from MongoDB Cloud Manager
+  api-key: "<b64 encoded api key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bdb-private-key
+  namespace: default
+type: Opaque
+data:
+  private.key: "<b64 encoded private key>"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-certs
+  namespace: default
+type: Opaque
+data:
+  mdb-instance.pem: "<b64 encoded, concatanated public and private keys>"
+  ca.pem: "<b64 encoded CA public key>" 
+  mdb-crl.pem: "<b64 encoded CRL data>" 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-mon-certs
+  namespace: default
+type: Opaque
+data:
+  mdb-mon-instance.pem: "<b64 encoded, concatanated public and private keys>"
+  ca.pem: "<b64 encoded CA public key>" 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mdb-bak-certs
+  namespace: default
+type: Opaque
+data:
+  mdb-bak-instance.pem: "<b64 encoded, concatanated public and private keys>"
+  ca.pem: "<b64 encoded CA public key>" 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bdb-certs
+  namespace: default
+type: Opaque
+data:
+  bdb-instance.pem: "<b64 encoded, concatanated public and private keys>"
+  ca.pem: "<b64 encoded CA public key>" 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: https-certs
+  namespace: default
+type: Opaque
+data:
+  cert.pem: "<b64 encoded HTTPS key>"
+  cert.key: "<b64 encoded HTTPS Signed Certificate or Certificate Chain>"
```

Comment:
I think the descriptions are mis-matched here. Also, I would say that `cert.key` is the b64 encoded HTTPS _private_ key.